src/uct/tcp/tcp_ep.c

```diff
@@ -714,16 +739,24 @@ ucs_status_t uct_tcp_ep_connect_to_ep(uct_ep_h tl_ep,
 
     if (!uct_tcp_cm_ep_accept_conn(ep)) {
         ucs_assert(ep->conn_state == UCT_TCP_EP_CONN_STATE_CLOSED);
-        /* EP that are created as CONNECT_TO_EP has to be full-duplex, set RX
+        /* EPs which are created as CONNECT_TO_EP has to be full-duplex, set RX
          * capability as well as TX (that's set in uct_tcp_ep_connect()) */
         uct_tcp_ep_add_ctx_cap(ep, UCT_TCP_EP_FLAG_CTX_TYPE_RX);
 
         uct_tcp_ep_ptr_map_del(ep);
 
         /* Use remote peer connection sequence number value, since the EP has to
-         * send the CONN_REQ to the peer has to find its EP in the EP PTR map */
+         * send the CONN_REQ to the peer which has to find its EP in the its EP
+         * PTR map */
```

Comment:
imo, previous comment was better, now you have double `its`

Reply:
fixed